modules/savanna/modules/graph-development/pages/load-data/load-from-blob.adoc

```diff
@@ -5,7 +5,7 @@ TigerGraph Savanna allows you to load data directly from Azure Blob Storage. Thi
 
 == 1) Select Source
 
-Once you’ve selected btn:[Azure Blob Storage], you will be asked to configure the Azure Blob Storage data source.
+.Once you’ve selected btn:[Azure Blob Storage], you will be asked to configure the Azure Blob Storage data source.
```

Comment:
```suggestion
Once you’ve selected btn:[Azure Blob Storage], you will be asked to configure the Azure Blob Storage data source.

```
@yulitg2110 I see what you are trying to fix, but adding a dot to this line is not the correct approach. You don't want this line to be numbered you want the next line ("Click on ...") to be numbered. The correct way to do that is to add a blank line.  A list should always start separated from the previous element by a blank line.